client/next.config.js

```diff
@@ -0,0 +1,2 @@
+const withCSS = require('@zeit/next-css');
+module.exports = withCSS();
```

Comment:
you forgot to add a line at the end of the file

Reply:
will add! thanks